controllers/dspipeline_controller.go

```diff
@@ -141,6 +142,14 @@ func (r *DSPipelineReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, err
 	}
 
+	// FixMe: Hack for stubbing gvk during tests as these are not populated by test suite
```

Comment:
Can you elaborate on this?

Reply:
Basically when the reconciler attempts to reconcile the resource it fails because the owner resoures (dsp resource) is returned without a group version kind (I don't know why) specified, this fails reconcilliation for tests. So taking inspiration from here, we [add this hack](https://github.com/operator-framework/operator-sdk/issues/727#issuecomment-581169171) to re-add them to allow tests to execute.

Reply:
Can you update the comment to link to the hack too? Thanks!

Reply:
#10

Reply:
I added a link to this discussion thread for full context.